src/app/session.service.ts

```diff
@@ -1,15 +1,37 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Http } from '@angular/http';
+import { GlobalSettingsService } from './globalSettings.service';
 
 @Injectable()
 export class SessionService {
 
-  constructor(private http: Http) {}
+  config;
```

Comment:
Shall we chat at the next code club about this? Might be one for more of an in depth discussion.